website/content/docs/secrets/gcp.mdx

```diff
@@ -72,11 +127,28 @@ management tool.
 
 1.  Alternatively, configure the audience claim value and the service account email to assume for plugin workload identity federation:
 
-    ```text
+<Tabs>
+
+<Tab heading="Vault CLI">
+
+   ```shell-session
```

Comment:
Remove indents. For example:

````
```shell-session
$ vault write gcp/config \
    identity_token_audience="<TOKEN AUDIENCE>" \
    service_account_email="<SERVICE ACCOUNT EMAIL>"
```
````